app/src/main/java/org/schabi/newpipe/local/subscription/dialog/FeedGroupDialog.kt

```diff
@@ -234,7 +235,8 @@ class FeedGroupDialog : DialogFragment() {
                     item.isSelected = isSelected
                     item.notifyChanged(PickerSubscriptionItem.UPDATE_SELECTED)
 
-                    val updateSelectedCountText = getString(R.string.feed_group_dialog_selection_count, this.selectedSubscriptions.size)
+                    subscriptionsCount = this.selectedSubscriptions.size
```

Comment:
A `val` should be used here as well, as keeping a reference to that variable outside is not necessary.